### ISSUE TITLE
ENHANCEMENT - more bosses

### ISSUE THREAD

**Issue by Doronimmo** [evanbowman/skyland-beta#109]
**Is your feature request related to a problem? Please describe.**
Currently, there is only one boss, and more bosses at the end of zones could help the difficulty curve (stop people from reaching the later zones undergeared via only stepping on natural encounters)

**Describe wanted behaviour**
1/more bosses at the end of zone 2 (this could also be a nice way to add lore)

**Describe alternatives you've considered**
current state is still very good.

**Comment by evanbowman** [comment]
Good idea! The zone exits having hostile encounters is meant to prevent people from reaching high levels without enough resources, but a more formal boss encounter would be fun and interesting!
